src/Adapter/Product/Image/ProductImagePathFactory.php

```diff
@@ -90,4 +136,20 @@ public function getHelperThumbnail(int $productId, int $shopId): string
     {
         return sprintf('%sproduct_mini_%d_%d.jpg', $this->temporaryImgDir, $productId, $shopId);
     }
+
+    /**
+     * @param Image $image
+     *
+     * @return string
+     */
+    private function getBaseImagePath(Image $image): string
```

Comment:
```suggestion
    private function getBaseImageFolderPath(Image $image): string
```

To avoid confusing it with all the other path functions in this service

Reply:
```suggestion
    private function getPathToImageDir(Image $image): string
```

So with the latest modifs Id go for this naming

Reply:
that would be misleading. It not only gets the dir, but also the original image of product without extension.